st2common/st2common/models/db/liveaction.py

```diff
@@ -59,6 +59,9 @@ class LiveActionDB(stormbase.StormFoundationDB):
     action = me.StringField(
         required=True,
         help_text='Reference to the action that has to be executed.')
+    is_action_workflow = me.BooleanField(
```

Comment:
Dude, is_workflow is better. is_action_workflow is confusing. Sorry for the nit :).